### ISSUE TITLE
Request: Larger and Bolder Notification Icon for Better Visibility

### ISSUE THREAD

**Issue by Ashpex** [gotify/android#391]
Hi,

I noticed that the current notification icon of Gotify is relatively small compared to other apps. Additionally, the thin outline of the icon makes it less recognizable.

![Screenshot_20241202_122727_Settings](https://github.com/user-attachments/assets/b7100ee4-be2d-4827-ac5a-fc41b4e7f4ee)

![Screenshot_20241202_122808_Niagara Launcher](https://github.com/user-attachments/assets/c240ef35-e48d-4040-82e0-66fdb9d39001)

I would like to suggest making the icon slightly larger and perhaps bolder to improve its visibility and recognition. I understand this may not be a critical issue, and you likely have more important priorities, but I thought it might be helpful to point it out.

Thank you.

**Comment by jmattheis** [comment]
Yeah, probably doesn't hurt when we expand the image content to the full width.

**Comment by Ashpex** [comment]
> Yeah, probably doesn't hurt when we expand the image content to the full width.

I wonder how expanding image content to full width is going to help the icon in this case?

**Comment by Ashpex** [comment]
> I mean this. Light gray is the actual image, and the icon doesn't fill the available space.
> 
> ![image](https://private-user-images.githubusercontent.com/14895212/393941390-03053dc7-6767-4b90-b763-1400f81a99d4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3ODYwNTIsIm5iZiI6MTczMzc4NTc1MiwicGF0aCI6Ii8xNDg5NTIxMi8zOTM5NDEzOTAtMDMwNTNkYzctNjc2Ny00YjkwLWI3NjMtMTQwMGY4MWE5OWQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA5VDIzMDkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhMzcwOGYwOTI1MjVmNDUyMjMyNjdmY2Q0ZjcyMGU0YWU4MjBhZmI0NWY5MTljY2Q1MjYwZTJjZmU1MGIxOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Xa6_VP6dZwr8tSVjgMFfv-m9W1LVM4ACXZmFEMnRrNM)
> 
> https://github.com/gotify/logo/blob/master/gotify-logo-notification.png

Ah, that probably would do it 😆
